mlflow/langchain/utils.py

```diff
@@ -278,13 +309,35 @@ def _validate_and_wrap_lc_model(lc_model, loader_fn):
     import langchain.llms.openai
     import langchain.schema
 
+    # lc_model is a file path
     if isinstance(lc_model, str):
-        if os.path.basename(os.path.abspath(lc_model)) != "chain.py":
+        if not os.path.exists(lc_model):
             raise mlflow.MlflowException.invalid_parameter_value(
-                f"If {lc_model} is a string, it must be the path to a file "
-                "named `chain.py` on the local filesystem."
+                f"If the provided model '{lc_model}' is a string, it must be a valid python "
+                "file path or a databricks notebook file path containing the code for defining "
+                "the chain instance."
             )
-        return lc_model
+
+        try:
+            with open(lc_model) as _:
```

Comment:
What is this `open` for?

Reply:
This is to determine if the lc_model is a normal file path or databricks notebook. I couldn't find a better way to do this, open to any suggestions.